src/lib/components/mods/ModGrid.svelte

```diff
@@ -197,6 +211,8 @@
       <FicsitCard fake />
     {/each}
   </div>
+{:else if $mods.error && $mods.error.message.includes("'Search' failed on the 'min' tag")}
+  {$t('search.failed.query-too-short')}
```

Comment:
The solution should be not sending queries that are invalid, rather than matching on the error message string

Reply:
The only condition that lets you see this error message is directly navigating to a page with a too-short query via URL editing. To make sure I'm understanding - editing line 31 to not perform the mods query is the preferred solution?

Reply:
Yea, this was just to display a helpful error message if you happened to go to a link that someone provided that was invalid already.